no_case: 0
setpoint: 0.0
option: "sine"